mu_repo/action_clone.py

```diff
@@ -0,0 +1,127 @@
+from mu_repo.execute_command import ExecuteCommand
+import os
+
+#===================================================================================================
+# Run
+#===================================================================================================
+def Run(params):
+    '''
+    mu-repo can deal with cloning a repository and other referenced repositories recursively.
+
+    To work this way, users are expected to commit the .mu_repo files they rely on and set
+    the base urls that users work with.
+
+    i.e.:
+
+    Say that someone is working with 3 projects: A, B and C and C depends on B and B depends on A
+    and they're all available on https://github.com/myuser
+
+    The urls in this case to be checked out would be something as:
+
+    ssh://git@github.com:myuser/projectA.git
+
+    https://github.com/myuser/projectA.git
+
+    So, the url: https://github.com/myuser or git@github.com:myuser has to be configured as the
+    'base' url for clones.
+
+    mu-repo in this case will work with git to provide that configuration, so, one should first
+    add that remote for mu-repo to work with:
+
+    git config --global --add mu-repo.remote-host ssh://git@github.com:myuser
```

Comment:
I'm thinking `mu-repo.remote` would be more align with the other options used by git, as `remote` is the term used through the standard git config:

```ini
[remote "origin"]
	url = https://bruno@eden.esss.com.br/stash/scm/esss/etk.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "fb-dist-12.0.18"]
	remote = origin
	merge = refs/heads/fb-dist-12.0.18
```